test/unitTests/OmnisharpPackageCreator.test.ts

```diff
@@ -194,4 +194,16 @@ suite('GetPackagesFromVersion : Gets the experimental omnisharp packages from a
         outPackages.length.should.equal(1);
         outPackages[0].experimentalPackageId.should.equal("win-x64");
     });
+});
+
+suite('GetVersionFilePackage : Gives the package for the latest file download', () => {
+    test('Contains the expected description', () => {
+        let testPackage = GetVersionFilePackage("someUrl", "somePath");
+        expect(testPackage.description).to.equal('Latest version information file');
+    }); 
+
+    test('Contains the url based on serverUrl and the pathInServer', () => {
+        let testPackage = GetVersionFilePackage("someUrl", "somePath");
+        expect(testPackage.url).to.equal('someUrl/somePath');
+    }); 
 });
```

Comment:
Are we able to provide an end-to-end integration test at this stage? something that will read the 'latest' property value, download a test version file, and based on that downloads a test O#? This can be a subsequent PR, but I'd like to get the full feature behind a test once we're ready, if it's reasonable...